security/keycloak/fuse-keycloak.adoc

```diff
@@ -174,7 +174,7 @@ Finally, ssh console has to switch the realm:
 ----
 karaf@root()> config:property-list --pid org.apache.karaf.shell
    completionMode = GLOBAL
-   hostKey = /data/servers/fuse-karaf-${version.org.jboss.fuse-karaf}/etc/host.key
+   hostKey = /data/servers/fuse-karaf-7.11.1.fuse-7_11_1-00005/etc/host.key
```

Comment:
is this correct? Will you have to switch it with each build in the future?

Reply:
thanks for spotting ;)